cost/kubecost/sizing/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v0.3
+
+- A new parameter for requesting recommendations "per cluster" or "overall" was added, if "per cluster" mode is configured, policy will discover clusters, iterate the clusters and request recommendations for each one. Previously, policy requested these recommendations in "overall" mode.
```

Comment:
This should be shortened and grammatical errors should be corrected. For example:

Added parameter to allow user to gather recommendations by cluster or across the entire account.

Reply:
@XOmniverse - Thank you, I fixed it!